### ISSUE TITLE
The Paciello Group comment on SC 2.6.1 Orientation

### ISSUE THREAD

**Issue by patrickhlauke** [w3c/wcag21#336]
The SC itself is fine, but the definition of "essential" is weak - the exceptions here are not strong enough. In all cases, an argument can be made that those examples quoted as exemptions could in fact be made to work in either orientation (e.g. in the case of the piano application, the piano keys could be split into two horizontal rows of keys when in portrait mode; the cheque example could allow for the image of the cheque to be zoomable/pannable in portrait mode). The only hurdle here is that yes, it would mean more work on the part of designers/developers to cater to a different aspect ratio/orientation, but it's not necessarily intrinsically *essential* to the content/activity to be shown in that particular way. So the exceptions here are mostly due to the fact that it would put undue (?) burden on developers? It may also be far too easy, looking at current wording, for a designer/developer to make something that only works in one orientation because they designed it that way and then claim that it's essential that it only work in that mode.
Also, term "essential" is too generic. Suggest the term defined should be a lot more specific, like "essential orientation". Lastly, since the normative SC text relies on on the definition of "essential", assuming that this makes the wording of the definition also normative - and for this reason, extra care should be taken to make sure it's not "handwavy" with regards to the exemptions (as noted above).

**Comment by steverep** [comment]
> The SC itself is fine, but the definition of "essential" is weak - the exceptions here are not strong enough. In all cases, an argument can be made that those examples quoted as exemptions could in fact be made to work in either orientation (e.g. in the case of the piano application, the piano keys could be split into two horizontal rows of keys when in portrait mode; the cheque example could allow for the image of the cheque to be zoomable/pannable in portrait mode).

@patrickhlauke, I strongly agree with this assessment that the current wording results in too weak a criterion.  The problem stems from the fact that the argument for exemptions is just an argument for having to lock the content, rather than an argument for why the content could not be viewed in either orientation as you stated.  I've argued for revised language to address this both [on the issue thread](https://github.com/w3c/wcag21/issues/70#issuecomment-316581421) and [on the mailing list](https://lists.w3.org/Archives/Public/w3c-wai-gl/2017AprJun/1132.html) unsuccessfully.  The language I proposed was the following, which takes the positive approach to requiring both views be possible and the exception comes from there:

**Orientation:** A mechanism is available to view content in all display orientations without loss of essential content or functionality, except where the display orientation is fixed by the user agent or is essential for usage or meaning.

> The only hurdle here is that yes, it would mean more work on the part of designers/developers to cater to a different aspect ratio/orientation, but it's not necessarily intrinsically essential to the content/activity to be shown in that particular way. So the exceptions here are mostly due to the fact that it would put undue (?) burden on developers?

IMHO, there is no big burden here and most responsive content will do just fine as is.  For cases where all content is meant to fit a display without scrolling, the author can do the work to rearrange things, or they can do nothing and allow the content to shrink using `vw` or `vh` units.  For example, the piano keys can just get smaller to maintain the same presentation.

> It may also be far too easy, looking at current wording, for a designer/developer to make something that only works in one orientation because they designed it that way and then claim that it's essential that it only work in that mode.

I think it goes beyond that.  I've tried to give the example of a game or other application that wants to use the device's gyroscope for a function (e.g. a driving or piloting game).  In that case, the content must be locked to avoid the OS making orientation switches, and the current SC stops there even though there's no reason the game couldn't be played in the other orientation.
